### PR TITLE
Sentiment classifier

### PR DESCRIPTION
**Pull request by kochsj**

### CODE REVIEW

**Comment by kochsj** [comment]
- [x] Utilizing a One-Hot Vector approach, create a Model that utilizes a Neural Network and has:
- [x] input layer
- [x] output layer
- [x] at least 2 Dropout/Dense layer pairs
- [x] Play with your activations to achieve the accuracy you would like, but discuss which ones you selected in your readme.
- [x] Train your Model!
- [x] Test your model with the validation set you put aside yesterday.
- [x] Save your trained Model as Json to be used again.
